apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js

```diff
@@ -39,7 +44,7 @@ class DomaMetersImporter extends AbstractMetersImporter {
         return {
             address: row[0],
             addressInfo: {
-                unitType: this.mappers.unitType[row[2].toLowerCase()],
+                unitType: this.mappers.unitType[String(row[2]).toLowerCase()],
                 unitName: row[1],
             },
             accountNumber: row[3],
```

Comment:
Is there or somewhere else in this class any mechanism of checking row length and corresponding data types? IMO looks quite dangerous

Reply:
This class receives `string[][]` data got from incoming file. But there was a erroneous file with cell below the main table. See description in the jira task.

Reply:
I do not think we should control string length here. It's better incapsulate it within models.

Reply:
Ok, sounds reasonable